v2/features/farms/FarmsTableRowBody.tsx

```diff
@@ -10,11 +13,39 @@ import { CoreSelectors } from "v2/store/core";
 import { UserData } from "picklefinance-core/lib/client/UserModel";
 import { UserSelectors } from "v2/store/user";
 import { getUserAssetDataWithPrices } from "./FarmsTableRowHeader";
+import { ChainNetwork } from "picklefinance-core";
+import { Network } from "../connection/networks";
+import { switchChain } from "../connection/ConnectionStatus";
 
 interface Props {
   jar: JarDefinition;
 }
 
+const ConnectButton: FC<{ network: Network | undefined }> = ({ network }) => {
+  const { library } = useWeb3React<Web3Provider>();
+  const allCore = useSelector(CoreSelectors.selectCore);
+
+  if (!network || !allCore || !library) return null;
+  return (
+    <Button onClick={() => switchChain(library, network.chainId, allCore)}>
+      <span>Connect to</span>
```

Comment:
@larrythecucumber321 can you please handle i18n here?

Reply:
done!